src/utils/isStandardSyntaxDeclaration.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
+
+/**
+ * Check whether a declaration is standard
+ *
+ * @param {import('postcss').Declaration | import('postcss-less').Declaration} decl
```

Comment:
I believe adding this would mean that postcss-less has to be a non-dev dependency, which we wouldn't want